src/object.rs

```diff
@@ -1541,7 +1541,7 @@ impl<T: ObjectType> ObjectExt for T {
         ::signal::connect_raw(
             self.as_object_ref().to_glib_none().0,
             signal_name.as_ptr() as *const _,
-            Some(mem::transmute(notify_trampoline::<Self, F> as usize)),
+            Some(*(&notify_trampoline::<Self, F> as *const _ as *const _)),
```

Comment:
Interesting how rust found type for first `_` in `as *const _ as *const _ `  😉

Reply:
Pointers are a weird thing in rust. You have to do it twice every time to convert between pointers of non primitives types.

Reply:
The first `as *const _` casts the `&T` into a `*const T` (there is only one option for `T` here, you can't do transitive casts from references). The second `as *const _` is from `*const T` to `*const glib_sys::GCallback` (which is what is required to be the final type).

Reply:
Thanks